### DISCUSSION TITLE
Selection manipulations

### DISCUSSION THREAD

**Discussion by PeterC-DLS** [silx-kit/h5web#1363] [Ideas]
It would be very useful to be able to manipulate selections using a pointing device.

I was thinking of adding drag handles to several positions of each selection shape (maybe visible when selected to be changed). Each handle would
1. move part of the shape such change an end point or translate an edge
2. with a modifier key rotate about the handle
An interior handle would translate the entire shape or rotate about its centre.

**Comment by axelboc** [comment]
As discussed last week, we're definitely intending to bring ROI features into H5Web. We're currently working on such a feature in a data acquisition application that uses H5Web. I recommend you do the same on your end (cf. https://github.com/silx-kit/h5web/discussions/1364#discussioncomment-4956184). Hopefully with two distinct use cases and implementations, we can identify the best abstractions to include in H5Web.
